cmd/limactl/copy.go

```diff
@@ -49,10 +50,24 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	arg0, err := exec.LookPath("scp")
+	useRsync, err := cmd.Flags().GetBool("rsync")
 	if err != nil {
 		return err
 	}
+
+	var arg0 string
+	if useRsync {
+		arg0, err = exec.LookPath("rsync")
+		if err != nil {
+			return err
+		}
+	} else {
+		arg0, err = exec.LookPath("scp")
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This duplicate the lookup code. We could do:

    tool := "scp"
    if useRsync {
        tool = "rsync"
    }

    arg0, err = exec.LookPath(tool)
    if err != nil {
        return err
	}

Reply:
Yeah, that makes sense. I was only following the discussion in the issue. I thought there was a consensus to go with having rsyn as a flag.